packages/ubuntu_bootstrap/pubspec.yaml

```diff
@@ -46,7 +46,11 @@ dependencies:
   ubuntu_localizations: ^0.3.3
   ubuntu_logger: ^0.1.0
   ubuntu_provision: ^0.1.0
-  ubuntu_service: ^0.2.2
+  ubuntu_service:
+    git:
+      url: https://github.com/spydon/ubuntu-flutter-plugins.git
+      ref: ubuntu_service/param-generics
+      path: packages/ubuntu_service
```

Comment:
Let's publish 0.3.0 first and use it here :)

Reply:
Updated to use the published version